src/main/kotlin/step4/racingcar/racingcar/CarRacing.kt

```diff
@@ -0,0 +1,37 @@
+package step4.racingcar.racingcar
+
+class CarRacing(private val racers: List<String>, private val moveCount: Int) {
+    private val _cars: MutableList<Car> = mutableListOf()
```

Comment:
```suggestion
    private val cars = racers.map { Car(it) }
```

execute 메서드를 호출할 때 가변인 _cars에게 넣어주는 것보다, racers를 받고 바로 넣어주면 좋지 않을까요?

Reply:
불필요한 동작이 많았네요~ 수정해두겠습니다